pom.xml

```diff
@@ -200,4 +181,40 @@
             </plugin>
         </plugins>
     </build>
+
+    <profiles>
+        <profile>
+            <id>Release</id>
+            <activation>
+                <activeByDefault>false</activeByDefault>
+                <property>
+                    <name>env.GITHUB_JOB</name>
+                    <value>build</value><!-- FIXME On purpose, because I want to be sure it failed -->
```

Comment:
Why do you do that?! Why don't you simply add `-PRelease` to the [maven command in the `release` build](https://github.com/gatling/gatling-maven-plugin/blob/main/.github/workflows/release.yml#L41)?

Reply:
Indeed, both work... I don't care which one.

Reply:
Yeah, I'd rather use the explicit profile setting please.